daemon/main.cpp

```diff
@@ -1391,7 +1391,6 @@ bool Daemon::handle_compile_done(Client *client)
         msg->user_msec = job_stat[JobStatistics::user_msec];
         msg->sys_msec = job_stat[JobStatistics::sys_msec];
         msg->pfaults = job_stat[JobStatistics::sys_pfaults];
-        end_status = job_stat[JobStatistics::exit_code];
```

Comment:
Why is it redundant?  IF the job exited with a status, use that, otherwise use 151 is how I read the code.

Reply:
IIUC, line 1389 sets end_status to the value of job_stat[JobStatistics::exit_code], then we set end_status to the same value here on line 1394 (job_stat[JobStatistics::exit_code] is not modified in the meantime).